dbms/src/Flash/Coprocessor/RemoteRequest.cpp

```diff
@@ -26,81 +26,110 @@ RemoteRequest RemoteRequest::build(
     const TiDBTableScan & table_scan,
     const TiDB::TableInfo & table_info,
     const PushDownFilter & push_down_filter,
-    const LoggerPtr & log)
+    const LoggerPtr & log,
+    Int64 physical_table_id,
+    bool is_disaggregated_compute_mode)
```

Comment:
How about using different methods for `is_disaggregated_compute_mode = false` and `is_disaggregated_compute_mode = true`?

Reply:
Done